src/tabletop_simulator/models/card.rb

```diff
@@ -0,0 +1,68 @@
+require "json"
+require "json-schema"
+
+# TODO: complete this schema
+CARD_SCHEMA = {
+  "type" => "object",
+  "required" => ["ObjectStates"],
+  "properties" => {
+    "ObjectStates" => {
+      "type" => "array",
+      "items" => {
+        "type" => "number"
+      },
+      "format" => "single-object-state"
+    }
+  }
+}
+
+JSON::Validator.register_format_validator(
+  "single-object-state",
+  ->(value) {
+    raise JSON::Schema::CustomFormatError.new("Multistate Cards not supported") unless value.length == 0
+  }
+)
```

Comment:
This'll be a nice tool for when we have to deal with massive ugly JSON blobs, it lets us detect failures early.

Following this PR I'm going to set up the rest of the schema and start to validate everytime we make a change. Then we won't get blindsided by having invalid JSON further down the pipe and have to figure out how it got that way.